test/signed-off-by-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+success="true"
+
+for commit in $(git cherry master | cut -d " " -f 2)
+do
+    if $(git show $commit | grep -q Signed-off-by ); then
```

Comment:
if we add -s to git show here, it wont emit the diff, which removes a bug where 'signed-off-by' can be in the diff.